### ISSUE TITLE
Compilation error when generating mocks on Mockative 2.1.0 - Expecting an element

### ISSUE THREAD

**Issue by vitorhugods** [mockative/mockative#88]
When using Mockative 2.1.0, the generated mock files look like this:

```kotlin
public fun mock(@Suppress("UNUSED_PARAMETER") type: KClass<NotificationTokenRepository>):
    NotificationTokenRepository = configure(NotificationTokenRepositoryMock()) { stubsUnitByDefault
    = true }
//  ^ Expecting an element
```

And this doesn't compile due to the `= true` being on the line below.

The equals sign _should_ at least be on the line above, or ideally like this:

```kotlin
public fun mock(@Suppress("UNUSED_PARAMETER") type: KClass<NotificationTokenRepository>):
    NotificationTokenRepository = configure(NotificationTokenRepositoryMock()) { 
        stubsUnitByDefault = true 
    }
```

This doesn't happen on Mockative `2.0.0` or `2.0.1`.

-----

I also want to dive deeper into this, but for now I'm sticking with `2.0.1` while I try to solve #87 first.

**Comment by vitorhugods** [comment]
Found the cause.

Fun fact:
Only happens when the interface/class name is 27 characters long 😄

Opening a PR

**Comment by vitorhugods** [comment]
Solved in #94

**Comment by vitorhugods** [comment]
@Nillerr do you have plans for a next release? Not wanting to pressure! Just want to know so I can plan a codebase migration to Mockative 2.x that were facing these two issues that are now fixed:
- #88
- #87

**Comment by Nillerr** [comment]
Hi @vitorhugods

I have released v2.2.0 just now. It should be available within the next hour.

Thank you for all your support ❤️
